certbot/tests/reporter_test.py

```diff
@@ -42,11 +39,9 @@ def test_no_tty_print_empty(self):
             self.reporter.print_messages()
         self.assertEqual(sys.stdout.getvalue(), "")
 
-    @mock.patch('certbot.reporter.os.getpid')
-    def test_atexit_print_messages(self, mock_getpid):
+    def test_atexit_print_messages(self):
         self._add_messages()
-        mock_getpid.return_value = self.rep_module.INITIAL_PID
-        self.reporter.atexit_print_messages()
+        self.reporter.print_messages()
```

Comment:
The problem with this is it reduces test coverage and this test was specifically written to test `atexit_print_messages` as suggested by the name.